modules/aruco/test/test_arucodetection.cpp

```diff
@@ -257,32 +280,45 @@ void CV_ArucoDetectionPerspective::run(int aprilDecimate) {
     Ptr<aruco::Dictionary> dictionary = aruco::getPredefinedDictionary(aruco::DICT_6X6_250);
 
     // detect from different positions
-    for(double distance = 0.1; distance <= 0.5; distance += 0.2) {
-        for(int pitch = 0; pitch < 360; pitch += 60) {
-            for(int yaw = 30; yaw <= 90; yaw += 50) {
+    for(double distance = 0.1; distance < 0.7; distance += 0.2) {
+        for(int pitch = 0; pitch < 360; pitch += (distance == 0.1? 60:180)) {
+            for(int yaw = 70; yaw <= 120; yaw += 40){
```

Comment:
> int yaw = 70; yaw <= 120; yaw += 40

yaw: 70, 110 ?

Old yaw values are: 30, 80

Reply:
even though the detection of a default marker happens using that yaw value, it was not the case for the inverted marker, however, it also occurs in the opposite way, that is, where the inverted marker is detected more easily than the default marker.